pyanaconda/modules/payloads/payload/live_image/installation.py

```diff
@@ -41,28 +41,39 @@
 class DownloadImageTask(Task):
```

Comment:
When the functionality changed this should be renamed? Maybe to something like `ProvideImageTask` or `GetImageTask`. It's not downloading anything for the local image.

Reply:
The main purpose of the task is to download an image. I don't want to hide that. It just ends earlier if the image is already "downloaded".